cam0:
  T_cam_imu:
    [0.9997598920056443, 0.021655048723997927, -0.0033492091040814645, 0.10655133459203663,
    0.021610170617149996, -0.9996832353474905, -0.012900755445729432, -0.020479009560267374,
    -0.0036275146807770832, 0.012825280891041978, -0.999911172708509, -0.011658972206891433,
    0.0, 0.0, 0.0, 1.0]
  camera_model: pinhole
  distortion_coeffs: [-0.01682, 0.00244, -0.00746, 0.00467]
  distortion_model: equidistant
  intrinsics: [608.22139, 608.45588, 485.46489, 400.81185]
  resolution: [960, 800]
  rostopic: /cam0/image_raw
cam1:
  T_cam_imu:
    [0.9997905920555996, 0.0204230183748785, -0.0012931966489092136, -0.09360051031320878,
    0.020393015960148354, -0.9995910858084147, -0.02004460207645495, -0.019942022813846254,
    -0.0017020391189715317, 0.020014032397636983, -0.9997982504335681, -0.0116839021194999,
    0.0, 0.0, 0.0, 1.0]
  T_cn_cnm1:
    [0.9999970959435367, 0.0012057793674637774, -0.002071742261018584, -0.200151,
    -0.001190940632536222, 0.9999737067076677, 0.007148808930372634, 0.000746693,
    0.002080307738981415, -0.007146321069627362, 0.9999722697806509, -0.000393262,
    0.0, 0.0, 0.0, 1.0]
  camera_model: pinhole
  distortion_coeffs: [-0.01274, -0.00994, 0.01027, -0.00543]
  distortion_model: equidistant
  intrinsics: [606.7294, 606.9206, 478.18754, 396.18382]
  resolution: [960, 800]
  rostopic: /cam1/image_raw
